### DISCUSSION TITLE
After loading 8.0, the following error will be prompted. The environment is 4.0

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#978] [Help]
警告	1	预定义类型“System.Runtime.CompilerServices.ExtensionAttribute”是在全局别名的多个程序集中定义的

**Comment by dlemstra** [comment]
Thanks for reporting this. It looks like you found something that I missed when removing direct support for .NET 4.0. I will publish a new release tonight to resolve this.

**Reply by dlemstra** [reply]
Can you try this again with the latest release?
